Assets/MRTK/Core/Services/MixedRealityToolkit.cs

```diff
@@ -81,11 +83,16 @@ public bool IsActiveInstance
         /// The public property of the Active Profile, ensuring events are raised on the change of the configuration
         /// </summary>
         /// <remarks>
-        /// When setting the ActiveProfile during runtime, the destroy of the currently running services will happen after the last LateUpdate()
+        /// <br>If changing the Active profile prior to the initialization (i.e. Awake()) of <see cref="MixedRealityToolkit"/> is desired, 
```

Comment:
Nit: We don't really use <br> in doccomments elsewhere, if only because they tend to make more noise when reading in code (even if they make the generated docs a little more useful)